pkgs/desktops/plasma-5/plasma-nm/default.nix

```diff
@@ -14,6 +14,10 @@ plasmaPackage {
       src = ./0001-mobile-broadband-provider-info-path.patch;
       inherit mobile_broadband_provider_info;
     })
+    (substituteAll {
+      src = ./0002-openvpn-binary-path.patch;
+      inherit openvpn;
+    })
```

Comment:
I would prefer `makeWrapper` to extend PATH actually, so we do not have to maintain this patch with future updates.